### PR TITLE
Add API feeder scripts

### PR DESCRIPTION
**Pull request by tekkac**
Adds a script to insert a list of contribution in the DB via the REST API.

### CODE REVIEW

File: scripts/feeder/contribution_feeder.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import csv
+from dotenv import load_dotenv
+
+
+CONTRIBUTIONS = "./scripts/feeder/contributions.csv"
+
+load_dotenv()
+API_KEY = os.environ["API_KEY"]
+GITHUB_KEY = os.environ["GITHUB_KEY"]
+HEADERS = {
```
Comment (ofux) [inline scripts/feeder/contribution_feeder.py:13]: looks like this constant is not used

File: scripts/feeder/contribution_feeder.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    "Accept": "application/json",
+    "Content-Type": "application/json",
+    "Api-Key": API_KEY
+}
+API_URL = "http://localhost:8000"
```
Comment (ofux) [inline scripts/feeder/contribution_feeder.py:18]:
```suggestion
API_URL = os.environ["API_URL"]
```

File: scripts/feeder/contribution_feeder.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    assert len(data) == 7
+    assert data[2] == 'github.com'
+    owner = data[3]
+    project = data[4]
+    issue = data[6]
+    return (owner, project, issue)
+
+GITHUB_API = "https://api.github.com/repos"
+
+def fetch_contribution_id(issue_url):
```
Comment (ofux) [inline scripts/feeder/contribution_feeder.py:98]: looks like this method is not used. Maybe simply remove it?

Comment (tekkac) [reply scripts/feeder/contribution_feeder.py:98]: yup, forgot to remove it from previous commit.

**Pull request merged**